cluster/manifests/skipper/deployment.yaml

```diff
@@ -3,16 +3,25 @@
 {{ $main_image := print "container-registry.zalando.net/teapot/skipper-internal:" "v0.21.211-1033" }}
 {{ $canary_image := "container-registry.zalando.net/teapot/skipper-internal:v0.21.212-1034" }}
 
-
 {{/* Optional canary arguments separated by "[cf724afc]" to allow whitespaces, e.g. "-foo=has a whitespace[cf724afc]-baz=qux" */}}
 {{ $canary_args := "" }}
 
+{{ $cluster_internal_cidrs := stringSlice "10.2.0.0/15" .Values.vpc_ipv4_cidr }}
```

Comment:
There already exists `list` template function, I think it should work instead of `stringSlice`

Reply:
See https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/825#discussion_r1790531050